client/src/containers/Topic/TopicList/TopicList.jsx

```diff
@@ -185,11 +186,16 @@ class TopicList extends Root {
       collapseConsumerGroups[topic.name] = false;
 
       this.getApi(uriTopicsGroups(selectedCluster, topic.name))
-        .then(value => {
-          tableTopics[topic.name].groupComponent = value.data
-          setState()
-        })
-    });
+          .then(value => {
+            tableTopics[topic.name].groupComponent = value.data
+            setState()
+          });
+      this.getApi(uriTopicLastRecord(selectedCluster, topic.name))
+          .then(value => {
+            tableTopics[topic.name].lastWrite = value.data.timestamp
```

Comment:
```suggestion
            tableTopics[topic.name].lastWrite = value.data.timestamp || ''
```
To remove loading box on empty topic